src/main/java/core/selenium/WebDriverConfig.java

```diff
@@ -22,12 +24,16 @@
  * @author Jesus Menacho
  * @version 1.0
  */
-public class WebDriverConfig {
+public final class WebDriverConfig {
     private String webBrowser;
     private String implicit;
     private String explicit;
     private static WebDriverConfig ourInstance = new WebDriverConfig();
 
+    /**
+     * Method let optain a instance.
+     * @return WebDriverConfig instance.
+     */
     public static WebDriverConfig getInstance() {
         return ourInstance;
     }
```

Comment:
write again your documentation at the bottom, must be written in third person

Reply:
That is write in third person.